torchvision/datasets/image.py

```diff
@@ -0,0 +1,46 @@
+import torch.utils.data as data
+from PIL import Image
+import os.path
+
+def make_dataset(file_name):
+    images = []
+    with open(file_name) as f:
```

Comment:
The file can be opened in read mode by calling `open(file_name, 'r')`, then you can remove the `read()` below